modules/auxiliary/spoof/llmnr/llmnr_response.rb

```diff
@@ -176,7 +174,10 @@ def should_print_reply?(host)
 
   def run
     check_pcaprub_loaded()
-    ::Socket.do_not_reverse_lookup = true
+    ::Socket.do_not_reverse_lookup = true  # Mac OS X workaround
+
+    # Avoid receiving extraneous traffic on our send socket
+    open_pcap({'FILTER' => 'ether host f0:f0:f0:f0:f0:f0'})
```

Comment:
Strike that, it's working if i use a real, wired interface. Fails on VMWare ifaces.